### ISSUE TITLE
[BUG] Trucking route not being set correctly, trailer and delivery markers not displaying

### ISSUE THREAD

**Issue by SirDornell** [xXJamieXx/paradox-bugs#147]
**Your Discord Name**
SirDornell#8747

**Describe the bug**
Driving to the Drop Off location the Highlighted Drop Off does not show up, and neither does the "E" prompt. Wiggling around the location and on multiple sides of the business does not prompt the Drop Off.

**To Reproduce**
[Steps to reproduce the bug]
1. Start Marina Truck Delivery
2. Drive to Location
3. Drop Off Highlight Doesn't Show Up

**Expected behavior**
Drop Off Location Highlighted

**Screenshots**
![image](https://user-images.githubusercontent.com/97981923/149999102-d328d225-3131-4b06-aa4b-a8cffbf1e5e2.png)

**Additional context**
[Add any other context about the problem here]

**Comment by Person-AL767** [comment]
FYI for anyone reading this- I experienced this once or twice with a couple other trucking spots- I noticed that when getting the truck, if the trailer doesn't have the glowing box under it at the depot, it's likely not going to show up at the location to drop the trailer off too. I have been able to fix this with a quick headpop- it's happened very very rarely.

Haven't confirmed with this spot yet, but can always try.

-Person__AL
